### ISSUE TITLE
`no-navigation-without-resolve`: Allow URL.href

### ISSUE THREAD

**Issue by marekdedic** [sveltejs/eslint-plugin-svelte#1487]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [x] I have tried restarting my IDE and the issue persists.
- [x] I have updated to the latest version of the packages.

### What version of ESLint are you using?

latest

### What version of `eslint-plugin-svelte` are you using?

master

### What did you do?

```svelte
<script>
  const one = new URL("https://svelte.dev");
  const two = new URL("/blog", "https://svelte.dev");
</script>

<a href={one.href}>One</a>
<a href={two.href}>Two</a>
<a href={one.toString()}>One</a>
<a href={two.toString()}>Two</a>
<a href={one}>One</a> <!-- Not 100% sure this calls toString -->
<a href={two}>Two</a> <!-- Not 100% sure this calls toString -->
```

### What did you expect to happen?

No error

### What actually happened?

Error

### Link to **GitHub Repo** with Minimal Reproducible Example

Will make a PR
